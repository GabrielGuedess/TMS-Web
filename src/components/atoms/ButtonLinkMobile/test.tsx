import { render } from '@testing-library/react';

import { ButtonLinkMobile } from '.';

describe('<ButtonLinkMobile />', () => {
  it('should render the heading', () => {
    const { container } = render(<ButtonLinkMobile href="/" title="gg" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
