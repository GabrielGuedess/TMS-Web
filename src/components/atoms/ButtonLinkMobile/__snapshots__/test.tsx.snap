// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonLinkMobile /> should render the heading 1`] = `
<a
  class="w-full p-4 text-center text-xl transition-all dark:hover:bg-shark-950 hover:bg-primary-400 text-comet-500 dark:text-dark-300 hover:text-white-lilac-50"
  href="/"
>
  gg
</a>
`;
